packages/sdk/client/src/packlets/proxies/halo-proxy.ts

```diff
@@ -58,15 +46,15 @@ export class HaloProxy implements Halo {
   private readonly _subscriptions = new EventSubscriptions();
   private readonly _devicesChanged = new Event<Device[]>();
   private readonly _contactsChanged = new Event<Contact[]>();
-  public readonly invitationsUpdate = new Event<CancellableInvitationObservable | void>();
-  public readonly identityChanged = new Event(); // TODO(burdon): Move into Identity object.
+  private readonly _invitationsUpdate = new Event<CancellableInvitationObservable[]>();
+  private readonly _identityChanged = new Event<Identity | undefined>(); // TODO(burdon): Move into Identity object.
 
-  private _invitations: CancellableInvitationObservable[] = [];
   private _invitationProxy?: DeviceInvitationsProxy;
 
-  private _identity?: Identity;
-  private _devices: Device[] = [];
-  private _contacts: Contact[] = [];
+  private _identity = new Observable<Identity | undefined>(undefined, this._identityChanged);
+  private _devices = new Observable<Device[]>([], this._devicesChanged);
+  private _contacts = new Observable<Contact[]>([], this._contactsChanged);
```

Comment:
We should really normalize (proto) Identity, Contact, and SpaceMember.
The common aspect is an identity
A Contact may have additional metadata.
A Member has more data about their status in the Space, but should reference a Contact.

Member > Contact > Identity